.circleci/config.yml

```diff
@@ -4,7 +4,7 @@
 #
 version: 2
 jobs:
-  build:
+  test:
     docker:
       # specify the version you desire here
       - image: circleci/node:8.12.0
```

Comment:
Please bump the node version to 10 here too (to be consistent with the release Node version).